helpers/integration.go

```diff
@@ -51,6 +51,40 @@ type HACBSTestResult struct {
 	Warnings  int    `json:"warnings"`
 }
 
+HACBSTestResult := `{
```

Comment:
```suggestion
HACBSTestResult = `{
```

Maybe this will solve the following GoSec error that we're seeing (based on [this post](https://stackoverflow.com/a/20508376)): 
```
helpers/integration.go:54:1: syntax error: non-declaration statement outside function body
[line 54 : column 1] - expected declaration, found HACBSTestResult
```

Reply:
Could we store the schema in a separate JSON Schema file and use that as a reference? That way we could also provide it in documentation and to the other teams if they need to do validation on their side.

Reply:
Fixed declaration and separated the pattern to it's own file, test_schema.json in most recent patch.